python/tk_3dsmax/maxscript.py

```diff
@@ -50,6 +50,9 @@ def create_menu(menu_name, menu_var):
         :param menu_name: String name of menu to create
         :param menu_var: MaxScript variable name in which the menu will be created
         """
+        # Remove old Shotgun menu entry from cache
+        MaxScript.unregister_menu("Shotgun")
```

Comment:
And this does not generate an error is the menu does not exist?

Reply:
The unregister_menu ask if the menu exists and it is True it deletes then.